addons/udes_stock_refactoring/tests/test_splitting.py

```diff
@@ -62,30 +56,62 @@ def _check_move_line(self, move_line, product, product_qty, package):
         self.assertEqual(move_line.package_id, package)
         self.assertEqual(move_line.result_package_id, package)
 
-    def _check_picking(
-        self, picking, state, group_name=False, location_id=False, location_dest_id=False
+    def _assert_picking_fields(
+        self,
+        picking,
+        state=None,
+        origin=None,
+        partner=None,
+        group_name=None,
+        location_id=None,
+        location_dest_id=None,
+        date_done=None,
```

Comment:
Could simplify this with just using kwargs
def _assert_picking_fields(self, picking, **kwargs):
    self.assertEqual(len(picking), 1)
    for key, val in kwargs.items():
        self.assertEqual(picking[key], val)

Reply:
Although this doesn't help with groups as names are being compared

Reply:
Yeah having to check groups by names made it a bit more awkward so felt it was better to use the named arguments for all fields being checked as there aren't that many.